manifests/vanilla/vsphere-csi-driver.yaml

```diff
@@ -499,7 +499,7 @@ spec:
             - name: LOGGER_LEVEL
               value: "PRODUCTION" # Options: DEVELOPMENT, PRODUCTION
             - name: GODEBUG
```

Comment:
Can remove GODEBUG env from Node Daemonset Pods?

Reply:
I had started this change yesterady but I'll keep this change to wcp only since you've submitted it for vanilla.